exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -589,8 +589,59 @@ func (p *Poloniex) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, er
 
 // GetOrderInfo returns order information based on order ID
 func (p *Poloniex) GetOrderInfo(orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+	orderInfo := order.Detail{
+		Exchange: p.Name,
+		Pair:     pair,
+	}
+
+	trades, err := p.GetAuthenticatedOrderTrades(orderID)
+	if err == nil {
```

Comment:
While I understand processing only on err nil (since Poloniex has a weird API), I think it would be nicer if we at least log the error. eg
```
trades, err := p.GetAuthenticatedOrderTrades(orderID)
err != nil && err.Error() != "Order not found, or you are not the person who placed it." {
   log.Error(log.Exchange, err)
}
```
as the type of error received could be of interest to us (eg the API has changed, the request is denied)
The for loop will be skipped on error anyway as there will be no trades in the slice to iterate over. We don't need to return on the error in case there are no trade details to return, but logging it will be helpful

Speaking of slices! Could you also change the for loop to be like `for i := range trades { ... ` as it is better for memory usage

Reply:
Yes, of course it's much better :)